README.md

```diff
@@ -21,44 +20,134 @@ This Connector implements the [Data Connector Spec](https://github.com/hasura/nd
 
-## For Hasura Users
+## Quickstart using the DDN CLI
 
-The Connector can be used with the DDN CLI. Use the `ddn dev` command to tell the CLI to automatically deploy, build and manage the Connector. Individual CLI Commands to build and deploy the Connector can be found [here](https://hasura.io/docs/3.0/cli/commands/)
+> [!TIP]
+> The following instructions are intended to get you quickly started with using the OpenAPI Lambda connector.
+> For further information on DDN, please check out the [Hasura DDN Getting Started Guide](https://hasura.io/docs/3.0/getting-started/overview/).
+
+> [!NOTE]
+> This section assumes that you have already setup a Supergraph and added a Subgraph.
+
+1. Initialize the connector
+
+```
+ddn connector init my_openapi --subgraph my_subgraph --hub-connector hasura/openapi
+```
+
+This will generate the necessary files into `my_subgraph/connector/my_openapi` directory. Supporting Typescript files for API calls will be created in this directory.
+
+2. Add the correct environment variables to `/my_subgraph/connector/my_openapi/.env.local`. Supported environment variables and their description are listed under [Supported Environment Variables](#supported-environment-variables) section.
+
+3. Modify the Docker container port in `my_subgraph/connector/my_openapi/docker-compose.my_openapi.yaml`. Typically, connectors default to port 8080. Each time you add a connector, please increment the published port by one to avoid port collisions. For example:
+
+```
+ports:
+  - mode: ingress
+    target: 8080
+    published: '8082'
+    protocol: tcp
+```
+
+4. Introspect the OpenAPI document using the connector
+
+```
+ddn connector introspect --connector my_subgraph/connector/my_openapi/connector.yaml
+```
+
+This will introspect your OpenAPI document and create an `api.ts` file, a `functions.ts` file and other supporting files required to run the Typescript project.
+
+- The `api.ts` file contains the Data Types and API calls from the OpenAPI document
+- The `functions.ts` file contains functions that wrap API calls. You can modify this `functions.ts` file to introduce business logic if you want to. See [Saving User Changes](#saving-user-changes) if you want to preserve your changes in this file when you introspect the OpenAPI document again.
+
+5. Add a [Data Connector Link](https://hasura.io/docs/3.0/supergraph-modeling/data-connector-links)
+
+```
+ddn connector-link add my_openapi --subgraph my_subgraph
+```
+
+This will create a file `my_subgraph/metadata/my_openapi.hml` that links your OpenAPI Connector to your Hasura Supergraph.
+
+6. Update the evironment variables listed in `my_subgraph/metadata/my_openapi.hml` (here, `MY_SUBGRAPH_MY_OPENAPI_READ_URL` and `MY_SUBGRAPH_MY_OPENAPI_WRITE_URL`) in `my_subgraph/.env.my_subgraph`.
+
+7. Start the connector in a Docker container
+
+```
+docker compose -f my_subpgraph/connector/my_openapi/docker-compose.my_openapi.yaml up
```

Comment:
should start all the services rather as this was added to the main docker-compose above right?

Reply:
This will only start the connector image, but we should add the steps for adding this image to the parent docker compose file and then the steps for running the full stack locally, per the READMEs for our other connectors: e.g., [Elasticsearch](https://github.com/hasura/ndc-elasticsearch/blob/main/README.md#6-include-the-connector-in-your-docker-compose)

Reply:
Resolved in [4572cc3](https://github.com/hasura/ndc-open-api-lambda/pull/41/commits/4572cc3e98b04f08ba08deb76bc1ccc0bba812b3)